test/test_ai_minimax.py

```diff
@@ -130,4 +131,16 @@ def testAIMinimaxTriesToBlockOpponentWinDiagonal(self):
 
         result = self.ai.move(self.board)
         expected_result = 9
-        self.assertEqual(result, expected_result, msg='\nRetrieved:\n{0} \nExpected:\n{1}'.format(result, expected_result))
\ No newline at end of file
+        self.assertEqual(result, expected_result, msg='\nRetrieved:\n{0} \nExpected:\n{1}'.format(result, expected_result))
+
+    def testAIMinimaxDoesNotMutateTheBoard(self):
```

Comment:
For this line:
https://github.com/hyoyou/tic-tac-toe-python/blob/master/tictactoe/ai_minimax.py#L34

This line looked a bit strange.  When I tried deleting it, the test suite still passed, but then when I tried running the game, I quickly discovered it was broken.  The board filled in with O's!

This is probably worth adding a test even after the fact.

Reply:
There may be a better way to do this.  Maybe clone the board or pass in a copy.